nncf/experimental/onnx/graph/nncf_graph_builder.py

```diff
@@ -47,15 +48,15 @@ def _is_valid_onnx_metatype(node: NodeProto) -> bool:
         """
         node_type = node.op_type
         metatype = ONNX_OPERATION_METATYPES.get_operator_metatype_by_op_name(node_type)
-        if metatype == ONNXConstantMetatype:  # We don't need to quantize Constants
-            nncf_logger.debug('The metatype is ONNXConstantMetatype, which means that the node is Constant.'
-                              'All constant nodes are not added to NNCFGraph. The node is skipped.')
+        if metatype == ONNXConstantMetatype:
+            # We don't need to quantize Constants
             return False
```

Comment:
Does it mean you build graph without constant but with constant sub-graphs?

Reply:
Constant is a node of ONNX opset, whose determines a constant value which is not weight. So, them typically go as attributes for some nodes. They make graph looks messy and dont impact on quantization process. And it seems they are not useful to be added in NNCFGraph for any algorithms al all.
But if some algorithms want to get the value from this constant it is till possible through ONNXGraph interface.

Reply:
Our agreement was to add all nodes as is and filter the insertion point graph to be able to apply QPS algorithm.

Reply:
Ok, I will remove the filtration

Reply:
It was not fixed.

Reply:
I will let you know when I update it. There is a need to update some harware fussing patterns to support elementwise ops with constant

Reply:
Done